### PR TITLE
Add a note to update the documentation when you add an instrumentation.

### PR DESCRIPTION
**Pull request by zippy1981**
The list of instrumentations in the docs is incomplete. Leaving a note here to add to that.

### CODE REVIEW

File: docs/development/AutomaticInstrumentation.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ Creating a new instrumentation implementation typically uses the following proce
    3. Select your branch from the drop down
    4. Click `Variables`, set `perform_comprehensive_testing` to true. (This is false for PRs by default for speed, but ensures your new code is tested against all the specified packages initially)
    5. Select `Stages To Run`, and select only the `build*`, `unit_test*` and `integration_test*` stages. This avoids using excessive resources, and will complete your build faster
-8. Once your test branch works, create a PR!
+   6. Add the instrumentation to the list of integrations in the [dotnet-core tracing documentation](https://docs.datadoghq.com/tracing/setup_overview/compatibility_requirements/dotnet-core/#integrations) and or [dotnet-framework tracing documentation](https://docs.datadoghq.com/tracing/setup_overview/compatibility_requirements/dotnet-framework/#integrations) as appropiate.
```
Comment (andrewlock) [inline docs/development/AutomaticInstrumentation.md:26]:
```suggestion
   6. Add the instrumentation to the list of integrations in the [dotnet-core tracing documentation](https://docs.datadoghq.com/tracing/setup_overview/compatibility_requirements/dotnet-core/#integrations) and/or [dotnet-framework tracing documentation](https://docs.datadoghq.com/tracing/setup_overview/compatibility_requirements/dotnet-framework/#integrations) as appropiate.
```

**Review by andrewlock — APPROVED** [review]
Looks good, thanks!

**Review by lucaspimentel — APPROVED** [review]

**Comment by zippy1981** [comment]
What's this waiting on?

**Comment by andrewlock** [comment]
Hi @zippy1981, sorry for the delay, we recently updated our CI build to not run on forks (for security reasons) but unfortunately that means also can't execute the necessary checks prior to merging, so the PR is in a blocked state 🙁 

We're trying to work out the best way forward now while keeping the protections in place, we'll merge this (and your other contributions!) as soon as we have a solution!

Thanks again, and sorry for the delay 🙁

**Comment by zippy1981** [comment]
Hello,

Since I'm simply editing. Readme.md, could we merge this without waiting on the tests that don't run on the forks

**Comment by zippy1981** [comment]
@andrewlock any progress on getting stuff so these can be merged?

**Comment by andrewlock** [comment]
Thanks @zippy1981 and @agocs - I have just merged this via #3080. Sorry for the delay, we're still trying to figure out the security aspects, but looks like Chris' approach works well. Thanks again both!

**Pull request closed without merging**